cli/ceph-disk

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/docker run --rm --privileged=true -v /etc/ceph:/etc/ceph -v /dev:/dev ceph/base ceph-disk "$@"
```

Comment:
I just managed to create OSDs with `ceph-disk` somehow using `-v /dev:/dev` instead of `-v /dev/sdb:/dev/sdb worked.`